Hugging Face for Education (#265)
huggingface/blog: education.md

> # **About Hugging Face for Education**

Reviewer: Before this, could you copy paste the section before "## Accelerate" from https://raw.githubusercontent.com/huggingface/blog/main/accelerate-library.md here?

You need to change the title and a couple of other places as well

> 

Reviewer: This will also need to be added at bottom of https://github.com/huggingface/blog/blob/main/_blog.yml

Author: done

As an Open Source company democratizing Machine Learning, [Hugging Face](https://huggingface.co/) believes it is essential to educate people from all backgrounds worldwide.

> Since March 2022 and the [ML demo.cratization tour](https://www.notion.so/ML-Demo-cratization-tour-with-66847a294abd4e9785e85663f5239652), experts from Hugging Face have taught hands-on classes on *How to Build Machine Learning Collaboratively* to more than 1000 students from 16 countries. As a second big step, we aim to teach machine learning to 1 million people by the end of 2023.

Reviewer: ```suggestion
Since March 2022 and the [ML demo.cratization tour](https://www.notion.so/ML-Demo-cratization-tour-with-66847a294abd4e9785e85663f5239652), experts from Hugging Face have taught hands-on classes on *How to Build Machine Learning Collaboratively* to more than 1000 students from 16 countries. Our new goal: we aim to teach Machine Learning to 1 million people by the end of 2023.
```

*This blog post describes what is behind Hugging Face for Education.*

## 🤗 **Education for All**

> 🗣️ Our goal is to make the potential and limitations of machine learning understandable to the greatest number of people. We believe that doing so will help evolve the field in a direction where the application of these technologies will lead to net benefits for society as a whole.

Reviewer: Let's capitalize Machine Learning here and in other places for consistency

Author: ok

Author: done

> 🗣️ Our goal is to make the potential and limitations of machine learning understandable to the greatest number of people. We believe that doing so will help evolve the field in a direction where the application of these technologies will lead to net benefits for society as a whole.

Reviewer: ```suggestion
🗣️ Our goal is to make the potential and limitations of machine learning understandable to everyone. We believe that doing so will help evolve the field in a direction where the application of these technologies will lead to net benefits for society as a whole.
```

- We describe in a very accessible way all [the uses of ML models](https://huggingface.co/tasks) (summarization, text generation, object detection…),
- We allow everyone to try all the ML models online without technical skills ([example](https://huggingface.co/cmarkea/distilcamembert-base-sentiment)),
- We warn against the harmful biases identified in some systems (like [GPT-2](https://huggingface.co/gpt2#limitations-and-bias)).
- We display open-source [ML apps](https://huggingface.co/spaces) and allow everyone to understand the potential of ML in one click.

## 🤗 **Education for Trainees**

🗣️ We want to allow software engineers to turn into machine learning engineers quickly with online learning, hands-on workshops, and other innovative techniques.

> - We provide [training courses](https://huggingface.co/course/chapter1/1) about natural language processing (NLP) and more domains (soon) using libraries from the Hugging Face ecosystem as well as the Hugging Face Hub. It’s completely free and without ads. The ultimate goal of this course is to learn how to apply Transformers to (almost) any machine learning problem!

Reviewer: Maybe clarify this is free (and using free tools), open sourced, and getting translated

Author: ok

Reviewer 2: 1 million is a great number. User experience matters as well!

- We provide a [book](https://transformersbook.com/), written by experts at Hugging Face, on Transformers and their applications to a wide range of NLP tasks.

## 🤗 **Education for Instructors**

🗣️ We want to empower educators with tools and offer collaborative spaces where students can build ML using open-source technologies and state-of-the-art ML models.

> - We’ve assembled [a toolkit](https://github.com/huggingface/education-toolkit) that instructors of Machine Learning or Data science can use to easily prepare labs, homework, or classes. The content is self-contained so that it can be easily incorporated into an existing curriculum. This content is free and uses well-known Open Source technologies (transformers, gradio, etc). Feel free to pick a tutorial and teach it!

Reviewer: ```suggestion
- We’ve assembled [a toolkit](https://github.com/huggingface/education-toolkit) that instructors of Machine Learning or Data Science can use to easily prepare labs, homework, or classes. The content is self-contained so that it can be easily incorporated into an existing curriculum. This content is free and uses well-known Open Source technologies (transformers, gradio, etc). Feel free to pick a tutorial and teach it!
```

    
    1️⃣ [A Tour through the Hugging Face Hub](https://github.com/huggingface/education-toolkit/blob/main/01_huggingface-hub-tour.md)
    
    2️⃣ [Build and Host Machine Learning Demos with Gradio & Hugging Face](https://colab.research.google.com/github/huggingface/education-toolkit/blob/main/02_ml-demos-with-gradio.ipynb)
    
    3️⃣ [Getting Started with Transformers](https://colab.research.google.com/github/huggingface/education-toolkit/blob/main/03_getting-started-with-transformers.ipynb)
    
- We're organizing a dedicated, free workshop (June 6) on how to teach our educational resources in your ML and data science classes. Do not hesitate to [register](https://www.eventbrite.com/e/how-to-teach-open-source-machine-learning-tools-tickets-310980931337).

> - We are currently doing a worldwide tour in collaboration with university instructors to teach more than 10000 students one of our core topics: How to build ML collaboratively? You can request someone on the Hugging Face team to run the session for your class via the [ML demo.cratization tour initiative](https://www.notion.so/ML-Demo-cratization-tour-with-66847a294abd4e9785e85663f5239652)**.**

Reviewer: Maybe let's put the image with all the talks we're doing

Author: good idea
<img width="535" alt="image" src="https://user-images.githubusercontent.com/95622912/164271167-58ec0115-dda1-4217-a308-9d4b2fbf86f5.png">

## 🤗 **Education Events & News**

**04/21**: ML demo.cratization tour in Florida. [Join](https://ufl.zoom.us/meeting/register/tJ0rduCqqD0oHNzysuD6Mfp_Tku5Q1SlFJ2g)

**06/06**: How to Teach Open-Source Machine Learning Tools. [Register](https://www.eventbrite.com/e/how-to-teach-open-source-machine-learning-tools-tickets-310980931337)

> 🔥 We are currently working on ambassadors programs, a new EDU offering, and a 101 ML course…! Stay tuned!

Reviewer: I would also link to the result of https://github.com/huggingface/blog/pull/267

Author: this will be done later

# **About Hugging Face for Education**

As an Open Source company democratizing Machine Learning, [Hugging Face](https://huggingface.co/) believes it is essential to educate people from all backgrounds worldwide.

> Since March 2022 and the [ML demo.cratization tour](https://www.notion.so/ML-Demo-cratization-tour-with-66847a294abd4e9785e85663f5239652), experts from Hugging Face have taught hands-on classes on *How to Build Machine Learning Collaboratively* to more than 1000 students from 16 countries. As a second big step, we aim to educate machine learning to 5 million people by the end of 2023.

Reviewer 3: nit:

```suggestion
Since March 2022 and the [ML demo.cratization tour](https://www.notion.so/ML-Demo-cratization-tour-with-66847a294abd4e9785e85663f5239652), experts from Hugging Face have taught hands-on classes on *How to Build Machine Learning Collaboratively* to more than 1000 students from 16 countries. As a second big step, we aim to teach machine learning to 5 million people by the end of 2023.
```

*This blog post describes what is behind Hugging Face for Education.*

> ## 🤗 **EDU for All**

Reviewer 3: Is the use of EDU here by design? One alternative would be "Education for All" and similar for the headings below, but no strong opinion from me here

Author: Both are very fine. no strong opinion as well.

Reviewer 2: hf.co/classroom
Above link does not work

<aside>

> 🗣️ We want to make technologies’ potential and limitations understandable to everyone and ensure that they evolve in a direction that benefits the greatest number of people.

Reviewer 3: A small suggestion - feel free to ignore if you want 

```suggestion
🗣️ Our goal is to make the potential and limitations of machine learning understandable to the greatest number of people. We believe that doing so will help evolve the field in a direction where the application of these technologies will lead to net benefits for society as a whole.
```

</aside>

- We describe in a very accessible way all [the uses of ML models](https://huggingface.co/tasks) (summarization, text generation, object detection…),
- We allow everyone to try all the ML models online without technical skills ([example](https://huggingface.co/cmarkea/distilcamembert-base-sentiment)),
- We warn against the harmful biases identified in some systems (like [GPT-2](https://huggingface.co/gpt2#limitations-and-bias)).
- We display open-source [ML apps](https://huggingface.co/spaces) and allow everyone to understand the potential of ML in one click.

## 🤗 **EDU for Trainees**

<aside>

> 🗣️ We want to allow software engineers to turn into machine learning engineers quickly with MOOCs, online learning, and more innovative techniques.

Reviewer 3: ```suggestion
🗣️ We want to allow software engineers to turn into machine learning engineers quickly with online learning, hands-on workshops, and other innovative techniques.
```

</aside>

- We provide [training courses](https://huggingface.co/course/chapter1/1) about natural language processing (NLP) and more domains (soon) using libraries from the Hugging Face ecosystem as well as the Hugging Face Hub. It’s completely free and without ads. The ultimate goal of this course is to learn how to apply Transformers to (almost) any machine learning problem!
- We provide a [book](https://transformersbook.com/), written by experts at Hugging Face, on Transformers and their applications to a wide range of NLP tasks.

## 🤗 **EDU for Educators**

<aside>

> 🗣️ We want to empower educators with tools and offer collaborative spaces where students can build ML using open-source technologies and SOTA ML models.

Reviewer 3: ```suggestion
🗣️ We want to empower educators with tools and offer collaborative spaces where students can build ML using open-source technologies and state-of-the-art ML models.
```

<h1>
    Introducing 🤗 Hugging Face for Education
</h1>
<div class="blog-metadata">
    <small>Published April 22, 2022.</small>
</div>
<div class="author-card">
    <a href="/VioletteLepercq">
        <img class="" src="">
        <div class="">
            <code>/VioletteLepercq</code>
            <span class="fullname">Violette Lepercq</span>
        </div>
    </a>
</div>

# **About Hugging Face for Education**

> As an Open Source company democratizing Machine Learning, [Hugging Face](https://huggingface.co/) believes it is essential to educate people from all backgrounds worldwide.

Reviewer 4: ```suggestion
As an open-source company democratizing machine learning, [Hugging Face](https://huggingface.co/) believes it is essential to educate people from all backgrounds worldwide.
```

> Since March 2022 and the [ML demo.cratization tour](https://www.notion.so/ML-Demo-cratization-tour-with-66847a294abd4e9785e85663f5239652), experts from Hugging Face have taught hands-on classes on *How to Build Machine Learning Collaboratively* to more than 1000 students from 16 countries. Our new goal: we aim to teach Machine Learning to 1 million people by the end of 2023.

Reviewer 4: ```suggestion
We launched the [ML demo.cratization tour](https://www.notion.so/ML-Demo-cratization-tour-with-66847a294abd4e9785e85663f5239652) in March 2022, where experts from Hugging Face taught hands-on classes on *How to Build Machine Learning Collaboratively* to more than 1000 students from 16 countries. Our new goal: to teach machine learning to 1 million people by the end of 2023.
```

> *This blog post describes what is behind Hugging Face for Education.*

Reviewer 4: ```suggestion
*This blog post describes what are doing to get there.*
```

## 🤗 **Education for All**

> 🗣️ Our goal is to make the potential and limitations of Machine Learning understandable to everyone. We believe that doing so will help evolve the field in a direction where the application of these technologies will lead to net benefits for society as a whole.

Reviewer 4: ```suggestion
🗣️ Our goal is to make the potential and limitations of machine learning understandable to everyone. We believe that doing so will help evolve the field in a direction where the application of these technologies will lead to net benefits for society as a whole.
```

- We describe in a very accessible way all [the uses of ML models](https://huggingface.co/tasks) (summarization, text generation, object detection…),
- We allow everyone to try all the ML models online without technical skills ([example](https://huggingface.co/cmarkea/distilcamembert-base-sentiment)),
- We warn against the harmful biases identified in some systems (like [GPT-2](https://huggingface.co/gpt2#limitations-and-bias)).
- We display open-source [ML apps](https://huggingface.co/spaces) and allow everyone to understand the potential of ML in one click.

> ## 🤗 **Education for Trainees**

Reviewer 4: ```suggestion
## 🤗 **Education for Students**
```

I think more people identify as "students" than "trainees"

> 🗣️ We want to allow software engineers to turn into Machine Learning engineers quickly with online learning, hands-on workshops, and other innovative techniques.

Reviewer 4: ```suggestion
🗣️ We want to lower the barrier to becoming a machine learning engineer with online learning, hands-on workshops, and other innovative techniques.
```
